doc/release/upcoming_changes/25721.deprecation.rst

```diff
@@ -0,0 +1,6 @@
+``np.linalg.lstsq`` now defaults to new ``rcond`` value
+-------------------------------------------------------
+`~numpy.linalg.lstsq` now uses the new rcond value of the machine precision
+times ``max(M, N)``.  Previously, the machine precision was used by a
```

Comment:
Some missing words here?

Reply:
The by was supposed to be a but, I think it's fine then?  (sorry force pushed those two changes)